src/chapters/14-provider-components.md

```diff
@@ -69,13 +69,38 @@ Let's start simple again and begin our refactor by creating a new template for o
 </div>
 ```
 
-All we're doing here is copy-pasting what was on our `index.hbs` page into our new component template.
+There is one minor change to note here: while extracting our markup into a component, we also renamed the `@model` argument to be `@rentals` instead, just in order to be a little more specific about what we're iterating over in our `#each` loop. Otherwise, all we're doing here is copy-pasting what was on our `index.hbs` page into our new component template. Now we just need to actually use our new component in the index template where we started this whole refactor! Let's render our `<Rentals>` component in our `index.hbs` template.
```

Comment:
```suggestion
There is one minor change to note here: while extracting our markup into a component, we also renamed the `@model` argument to be `@rentals` instead, just in order to be a little more specific about what we're iterating over in our `{{#each}}` loop. Otherwise, all we're doing here is copy-pasting what was on our `index.hbs` page into our new component template. Now we just need to actually use our new component in the index template where we started this whole refactor! Let's render our `<Rentals>` component in our `index.hbs` template.
```

I think that's how are referring to it in the other chapters.

Reply:
I don't get this suggestion...looks like nothing changed?

Reply:
oh i see it now. these suggestion things make it so hard to difficult to parse small diffs 😢